class.jetpack-data.php

```diff
@@ -99,6 +99,8 @@ public static function is_usable_domain( $domain, $extra = array() ) {
 			return true;
 		}
 
+		return true;
```

Comment:
This would keep us from running the next several lines in this method. Perhaps you short-circuited this in testing and forgot to remove it?